data-connection/src/main/java/io/micronaut/data/connection/annotation/Connectable.java

```diff
@@ -83,4 +83,29 @@
      */
     boolean readOnly() default false;
 
+    /**
```

Comment:
Since there is `ConnectableInterceptor` that is creating custom `ConnectionDefinition` I thought we could use this annotation for this purpose so users can customize module and action that will be traced. Don't see other way than use this annotation for the repository that needs to be traced. Without it, we cannot obtain details about class/method being executed when connection is established.